packages/scheduler/src/forks/SchedulerMock.js

```diff
@@ -637,8 +641,14 @@ export {
   unstable_advanceTime,
   reset,
   setDisableYieldValue as unstable_setDisableYieldValue,
+  forceYield as unstable_forceYield,
 };
 
+let forcedYield = false;
+function forceYield() {
+  forcedYield = true;
+}
```

Comment:
There is probably a better way to do this - but I couldn't figure out how to force this `SchedulerMock` to immediately yield the work.